utils/docker/Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM python:3
+FROM alpine:latest
 
-SHELL ["/bin/bash", "-c"]
+RUN apk upgrade --no-cache && \
+    apk add --update --no-cache bash sudo git nodejs npm fzf ranger curl python3-dev py3-pip neovim cargo lazygit g++
 
-ENV DEBIAN_FRONTEND=noninteractive
-
-COPY script.sh /tmp/script.sh
-
-RUN bash -c "/tmp/script.sh"
+RUN LVBRANCH=master curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh | bash -s -- -y
```

Comment:
Shouldn't we try to match the version ran in the container with the current branch ? I think that's why the setup script was copied

Reply:
Well it did not do that previously. The script just installed whatever was in the `master` branch. Only the script was copied in and not the actual LVIM git repo.

I just pushed a commit which uses a container build argument to point it to correct `LVBRANCH`. This way whoever is building the container will be able to point it at a branch on GitHub.

The real question here is this meant to be a development container? Then copying in the local git repo makes sense.

Or is it a user container? Then just pointing it to a branch on GitHub makes sense.

Reply:
Would you mind moving the changes to the dockerfile to a separate PR? I have a feeling it's gonna take some elbow grease 😶